examples/tacopie/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2016, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-tacopie)
+
+# download tacopie
+hunter_add_package(tacopie)
+
+# now tacopie can be used
+find_package(tacopie CONFIG REQUIRED)
```

Comment:
I don't see `tacopieConfig.cmake` installed, so this will not work. Please follow links from this part to create one:
* https://docs.hunter.sh/en/latest/creating-new/cmake.html#important